frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/TemporalBucketPickerPopover.tsx

```diff
@@ -8,22 +8,22 @@ import {
   getBucketListItem,
 } from "./BaseBucketPickerPopover";
 
-function checkBucketIsSelected(item: BucketListItem) {
-  return !!item.selected;
-}
-
 export function TemporalBucketPickerPopover({
   query,
   stageIndex,
   column,
   buckets,
+  isEditing,
   onSelect,
   ...props
 }: CommonBucketPickerProps) {
   const selectedBucket = useMemo(() => Lib.temporalBucket(column), [column]);
 
   const items = useMemo(
-    () => buckets.map(bucket => getBucketListItem(query, stageIndex, bucket)),
+    () => [
+      ...buckets.map(bucket => getBucketListItem(query, stageIndex, bucket)),
+      { displayName: t`Don't bin`, bucket: null },
```

Comment:
I think we could set `selected: !selectedBucket` here and avoid the check in `checkBucketIsSelected`

Reply:
fixed. I'm not 100% sure we need memoization here, but if `getBucketListItem` is slow, maybe it makes sense.

check on isEditing is required for the initial rendering (Don't bin is preselected even if we didn't add Created At to the breakout)